source/common/config/well_known_names.cc

```diff
@@ -76,8 +76,8 @@ TagNameValues::TagNameValues() {
   // mongo.[<stat_prefix>.]cmd.(<cmd>.)*
   addTokenized(MONGO_CMD, "mongo.*.cmd.$.**");
 
-  // cluster.[<route_target_cluster>.]grpc.[<grpc_service>.](<grpc_method>.)*
-  addTokenized(GRPC_BRIDGE_METHOD, "cluster.*.grpc.*.$.**");
+  // cluster.[<route_target_cluster>.]grpc.[<grpc_service>.].(<grpc_method_name>.)[<stat_name>]
+  addRe2(GRPC_BRIDGE_METHOD, R"(^cluster\..*?\.grpc\..*\.((.*)\.))");
```

Comment:
prior to the conversion of this particular extractor to tokenized form, it first looked like this (back when we used std::regex):
```
  // cluster.[<route_target_cluster>.]grpc.[<grpc_service>.](<grpc_method>.)<base_stat>
  addRegex(GRPC_BRIDGE_METHOD, R"(^cluster(?=\.).*?\.grpc(?=\.).*\.((.*?)\.)\w+?$)", ".grpc.");
```
and then this:
```
  // cluster.[<route_target_cluster>.]grpc.[<grpc_service>.](<grpc_method>.)*
  addRe2(GRPC_BRIDGE_METHOD, R"(^cluster\.<NAME>\.grpc\.<NAME>\.((<NAME>)\.))", ".grpc.");
```

Would the prior Re2 version pass your new test? I'm guessing it won't but I thought I'd ask.